extensions.json

```
@@ -477,6 +477,11 @@
       "id": "fabiospampinato.vscode-todo-plus",
       "repository": "https://github.com/fabiospampinato/vscode-todo-plus"
     },
+    {
+      "id": "fbaligand.vscode-logstash-editor",
+      "repository": "https://github.com/fbaligand/vscode-logstash-editor",
+      "version": "0.4.0"
```

When pinning a specific version, please also check out the corresponding release tag:

```suggestion
      "version": "0.4.0",
      "checkout": "v0.4.0"
```

Hint: Running `node add-extension https://github.com/fbaligand/vscode-logstash-editor --checkout` can do this automatically for you.